### PR TITLE
Update translation: pt-BR, pt-PT

### PR DESCRIPTION
**Pull request by LuanRoger**
**Details of Changes**
- Updated pt-BR, pt-PT language

**Validation**
- [x] Built and ran the app.

### CODE REVIEW

**Review by tsvietOK — CHANGES_REQUESTED** [review]
Looks like you modified `Resources.resw` file manually. All changes should be done only in `Files.pt-BR.xlf`. In PR include only .xlf file. For more details see our [documentation](https://files-community.github.io/docs/#/articles/translating-files).

**Comment by LuanRoger** [comment]
@tsvietOK I didn't modify the file manually, but I think I know why it is as modified:
I created an issue (#4402) explaining that the language was not changing from PT-PT to PT-BR, but it is really changing the language but some strings are the same, which was not meant to be.
When I did the merge of the upstream for my fork I chose not to bring these two files because I had already worked with them, so the only thing that should have been modified would be Files.pt-BR.xlf, but as I explained earlier this does not happen.

**Comment by tsvietOK** [comment]
I recommend you to exclude Resources file from this PR, since it will be regenerated later.

**Comment by tsvietOK** [comment]
I guess the problem is here. (target-language)
https://github.com/files-community/Files/blob/cd2f0355bf8bfabe18c31647ca49a0d5cff79763/Files/MultilingualResources/Files.pt-PT.xlf#L3

File: Files/MultilingualResources/Files.pt-BR.xlf [hunk 1]
```diff
@@ -1184,12 +1191,12 @@
           <target state="translated">Ver perfis de amostra</target>
         </trans-unit>
         <trans-unit id="SettingsPreferencesAppLanguage.Header" translate="yes" xml:space="preserve">
-          <source>App language</source>
-          <target state="translated">Idioma do aplicativo</target>
+          <source>App Language</source>
```
Comment (tsvietOK) [inline Files/MultilingualResources/Files.pt-BR.xlf:1194]: By the way, don't change string in `source` tag, the value is taken from Strings/en-US/Resources.resw file. Just change translation in `target` tag.

**Review by tsvietOK — APPROVED** [review]

**Pull request merged**